tests/php/TestPlugin.php

```diff
@@ -1,21 +1,28 @@
 <?php
+/**
+ * Tests for Plugin class.
+ *
+ * @package BlockScaffolding
+ */
 
-namespace XWP\BlockExtendTest;
+namespace XWP\BlockScaffolding;
 
+use Mockery;
 use WP_Mock;
-use XWP\BlockExtend\Plugin;
 
 /**
  * Test the WordPress plugin abstraction.
  */
-class PluginTest extends BlockExtendTestCase {
+class TestPlugin extends WP_Mock\Tools\TestCase {
```

Comment:
Having our own parent test case would allow us to add various helpers, etc. It also allows us to keep the shared bootstrap logic such as `use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;` into a dedicated file without having to replicate it for every script.